src/open_producten/producttypen/admin/prijs.py

```diff
@@ -31,6 +32,13 @@ class PrijsOptieInline(admin.TabularInline):
 class PrijsAdmin(admin.ModelAdmin):
     model = Prijs
     inlines = [PrijsOptieInline]
+    list_display = ("__str__", "actief_vanaf")
+    list_filter = ("product_type__naam", "actief_vanaf")
 
     def get_queryset(self, request):
         return super().get_queryset(request).select_related("product_type")
+
+    def has_change_permission(self, request, obj=None):
+        if obj and obj.actief_vanaf < datetime.today().date():
+            return False
+        return super().has_change_permission(request, obj)
```

Comment:
I still feel that this will come up as an issue in the future, but tomorrows problems are for tomorrow's me.

And by me I mean you.